### DISCUSSION TITLE
Outputting data on change in appearance

### DISCUSSION THREAD

**Discussion by walsmanjason** [mooch443/trex#212] [Questions] [answered]
I have a thermal video of an individual in an area and want to track both the individual's video and how dark they are, as a measure of their body temperature. I know Trex looks at how dark an individual is compared to background when training IDs. Is there an easy way to output this information? Going one step further, is there a way to ouput this information over the course of the video to see how dark the individual is at different time points in the video?

**Comment by mooch443** [comment, marked as answer]
Hey, oh wow I seem to have forgotten about the Discussions tab! Essentially what you were asking (in case it's still relevant) is whether you can output a stream of difference images? Yes:
```
-tracklet_export_difference_images true
-tracklet_max_images 0
-tracklet_normalize_orientation true/false
-output_image_per_tracklet true
```

And then when you export your data (pressing S or using ``-auto_quit``) it will take a long time to export image streams as well.

**Reply by walsmanjason** [reply]
Thank you!
